go.mod

```diff
@@ -36,16 +36,16 @@ require (
 	go.opentelemetry.io/otel/sdk/metric v0.34.0
 	go.opentelemetry.io/otel/trace v1.11.2
 	golang.org/x/crypto v0.5.0
-	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
+	golang.org/x/mod v0.6.0
 	golang.org/x/text v0.6.0
-	helm.sh/helm/v3 v3.10.3
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/client-go v0.25.4
-	k8s.io/component-base v0.25.2
+	helm.sh/helm/v3 v3.11.0
+	k8s.io/api v0.26.0
```

Comment:
we can't bump this yet because of CAPI 

ref @jont828 https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/pull/13#issuecomment-1400867269

Reply:
/hold